src/scss/2022/components/_hero.scss

```diff
@@ -49,25 +54,32 @@
     font-weight: 100;
     font-stretch: 80%;
     // Allows the font to be fluid in relation to the hero size
-    font-size: min(24vw, calc(var(--hero-max-width) * 0.24));
     text-align: right;
     grid-area: d;
     align-self: end;
     color: var(--color-gray);
     position: relative;
     z-index: 200;
+    
+    font-size: min(20vw, calc(var(--hero-max-width) * 0.2));
+    
+    @media (min-width: mq(sm)) {
+      font-size: min(24vw, calc(var(--hero-max-width) * 0.24));
+    }
   }
 
   &__edition {
     font-weight: 200;
     // Allows the font to be fluid in relation to the hero size
-    font-size: min(25vw, calc(var(--hero-max-width) * 0.025));
+    font-size: min(5vw, calc(var(--hero-max-width) * 0.025));
     grid-area: c;
     align-self: center;
     color: var(--color-gray);
   }
   
   &__logo {
+    width: 40vw;
```

Comment:
I'm good with this and the max-width values not having a var name as long as it's just used in one place!